packages/ai-jsx/src/batteries/docs.tsx

```diff
@@ -635,27 +635,31 @@ export class FixieCorpus<ChunkMetadata extends Jsonifiable = Jsonifiable> implem
   }
 
   async search(query: string, params?: { limit?: number }): Promise<ScoredChunk<ChunkMetadata>[]> {
-    const response = await fetch(`${this.fixieApiUrl}/corpora/${this.corpusId}:query`, {
+    const url = `${this.fixieApiUrl}/corpora/${this.corpusId}:query`;
+    const response = await fetch(url, {
       method: 'POST',
       headers: {
         'Content-Type': 'application/json',
         Authorization: `Bearer ${this.fixieApiKey}`,
       },
-      body: JSON.stringify({ query_string: query, chunk_limit: params?.limit }),
+      body: JSON.stringify({
+        corpus_id: this.corpusId,
+        query,
+        page_size: params?.limit ?? 10,
+      }),
     });
     if (response.status !== 200) {
       throw new AIJSXError(
-        `Fixie API returned status ${response.status}: ${await response.text()}`,
+        `Fixie API call to ${url} returned status ${response.status}: ${await response.text()}`,
         ErrorCode.FixieStatusNotOk,
         'runtime'
       );
     }
-    const apiResults = await response.json();
-    return apiResults.chunks.map((result: any) => ({
+    const apiResult = await response.json();
+    return apiResult.results.map((result: any) => ({
       chunk: {
-        content: result.content,
-        metadata: result.metadata,
-        documentName: result.document_name,
+        content: result.snippet,
+        documentName: result.document_id,
```

Comment:
This is not a document name.  I think we should probably only surface the chunk content for now and add more fields once we have some translation between DB models and the API we're actually serving.